modules/gdscript/gdscript_disassembler.cpp

```diff
@@ -978,6 +1079,12 @@ void GDScriptFunction::disassemble(const Vector<String> &p_code_lines) const {
 
 				incr += 5;
 			} break;
+			case OPCODE_RETURN_TYPED_DICTIONARY: {
+				text += "return typed dictionary ";
+				text += DADDR(1);
+
+				incr += 8;
+			} break;
```

Comment:
Some typed array/dictionary opcodes don't have type disassembly. No need to fix now, just a note for future PR.